packages/mattermost-redux/src/selectors/entities/teams.ts

```diff
@@ -333,6 +333,15 @@ export const getChannelDrawerBadgeCount: (state: GlobalState) => number = create
     },
 );
 
+export const isTeamSameWithCurrentTeam: (state: GlobalState, teamName: string) => boolean = createSelector(
```

Comment:
Also, `isTeamSameWithCurrentTeam` shouldn't use `createSelector` because it takes an argument. With an argument, that means that two places calling it with different values will overwrite each others saved results making memoization pointless. This also isn't performance-sensitive enough that memoization is really needed

Reply:
Thank you. I made a mistake.
I removed `createSelector`.

Reply:
No worries at all. That's a pattern that caused us a lot of problems in the past, so I've gotten good at tracking it down 😄